src/lib/utils/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './query-builder';
```

Comment:
I was first questioning whether we really need this file in the first place, but looking at other utility packages written in typescript this seems to be the good practice. TIL.

I do have one tiny nitpick that can make things a bit smoother. What do you think of doing this export?
```ts
export * as QueryBuilder from './query-builder';
```

In this way, when importing in `resources.ts` we can do something like this:
```ts
import { QueryBuilder } from "./utils";
```

This, from my perspective, brings two advanteages:
- Narrow down what we import from the `utils` package/module (don't know the exact terminology in TypeScript) in case `utils` expands in the future.
- When importing from utils now, we know exactly what we get from it, instead of "guessing" with `* as QueryBuilder`.